<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelezionaMag</class>
 <widget class="QDialog" name="SelezionaMag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>124</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>334</width>
       <height>73</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>334</width>
       <height>73</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>334</width>
       <height>73</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>334</width>
       <height>73</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Seleziona magazzino</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="magCombo">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>101</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="button_ok">
     <property name="text">
      <string>Rendi predefinito</string>
     </property>
     <property name="icon">
      <iconset resource="../fabaria.qrc">
       <normaloff>:/images/predefinito.png</normaloff>:/images/predefinito.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="button_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset resource="../fabaria.qrc">
       <normaloff>:/images/dialog-close.png</normaloff>:/images/dialog-close.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../fabaria.qrc"/>
 </resources>
 <connections/>
</ui>
